src/test/component/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.9)
 
-file(TO_NATIVE_PATH ${CMAKE_BINARY_DIR}/tool/cpp/cppwinrt/cppwinrt.exe cpp_exe)
+file(TO_NATIVE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../tool/cppwinrt/build/cppwinrt.exe cpp_exe)
```

Comment:
Seems like a bad idea. The CMake binary directory is set independently of the src directory structure.

Reply:
This is intentional. The cppwinrt binaries all build to a separate folder optimized for the developer inner loop.

Reply:
Why can't this separate folder live under CMAKE_BINARY_DIR?

Reply:
The cmake binary dir is very inconvenient for dev inner loop.

Reply:
I could of course make the cmake build default to the CMAKE_BINARY_DIR while the VS build uses a more intuitive location for local builds.

Reply:
done